apps/server/src/infra/sync/tsp/tsp-sync.service.ts

```diff
@@ -98,6 +149,120 @@ export class TspSyncService {
 		return savedSchool;
 	}
 
+	public mapTspDataToOauthData(
```

Comment:
Put this method into an extra mapper class.

Reply:
done